apps/estuary/lib/estuary/message_handler.ex

```diff
@@ -5,18 +5,26 @@ defmodule Estuary.MessageHandler do
   use Elsa.Consumer.MessageHandler
   alias Estuary.DataWriter
 
+  @updated_event_stream "updated_event_stream"
+
   def handle_messages(messages) do
     messages
     |> Enum.map(fn message ->
       message.value
       |> Jason.decode!()
+      |> broadcast_event()
```

Comment:
One alternative here would be to send the batch of events to the live view via the broadcast. However, I think that there will probably be few enough of them that one at a time is probably fine for now.